app/scripts/contentscript.js

```diff
@@ -319,26 +336,25 @@ const destroyLegacyExtensionStreams = () => {
   legacyPagePublicConfigChannel.removeAllListeners();
 
   legacyExtMux.removeAllListeners();
-  legacyExtMux.destroy();
-
   legacyExtChannel.removeAllListeners();
-  legacyExtChannel.destroy();
-
   legacyExtPublicConfigChannel.removeAllListeners();
+
+  legacyExtMux.destroy();
+  legacyExtChannel.destroy();
   legacyExtPublicConfigChannel.destroy();
```

Comment:
Is moving the .destroy calls to the end of these sequences foundational to this fix or a cleanup item?

Reply:
good question. No. This is cleanup. I will update the description @brad-decker